en/examples/README.md

```diff
@@ -54,8 +54,21 @@ See [QGroundControl > Download and Install](https://docs.qgroundcontrol.com/en/g
 To build the examples follow the instructions below, replacing *takeoff_and_land* with the name of the specific example.
 Any exceptions will be covered in the page for the associated example(s).
 
+
 #### Linux
 
+Ubuntu or Fedora users should install the MAVSDK C++ _*.deb_ or _*.rpm_ packages from the [Github release page](https://github.com/mavlink/MAVSDK/releases) in the normal way (or [build and install the SDK C++ Library](../contributing/build.md#sdk_system_wide_install) system-wide).
+
+Then build the example:
+```sh
+cd example/takeoff_land/
+mkdir build && cd build
+cmake ..
+make
+```
+
+#### macOS

```

Comment:
@julianoes Note, added macOS section, because I presume this cannot use rpm or deb files.

Reply:
Ok, we need to add support via brew soon.